using/api/pie-controller.md

```diff
@@ -0,0 +1,65 @@
+# PieController 
+
+## ES5 access 
+When you load the controller.js in a script tag, a var _pie_controller is created, to give you access to the controller.
+
+ ```
+   <script scr="controller.js"></script>
+   <script>
+     console.log(_pie_controller);
+     
+     //output 
+     [object object] 
+   </script>
+   
+ ```
+   
+
+## Javascript Api
+ 
+### Methods
+ 
+#### setConfig(json)
+Set the configuration object for the pies
+
+ ```
+  _pie_controller.setConfig({
```

Comment:
i'd prefer that the model/config is set in the constructor of the class definition - not set.